core/src/androidMain/kotlin/gatt/Callback.kt

```diff
@@ -174,3 +181,9 @@ private fun Int.toStatus(): State.Disconnected.Status? = when (this) {
     GATT_CONN_CANCEL -> Cancelled
     else -> Unknown(this)
 }
+
+private fun <E> SendChannel<E>.trySendOrLog(element: E) {
+    trySend(element).getOrElse {
+        Log.w(TAG, "Callback was unable to deliver $element")
```

Comment:
Is there any use in also printing the exception here?

Reply:
Ya, you're probably right. I was thinking this condition shouldn't occur. I think it would only occur if the OS delivered us a response _after_ a disconnected state, which would be very bizarre for the OS to do.

Being that we don't expect it, if it is seen, it would be good a stacktrace to see where the message is coming from along with the message.